README.md

```diff
@@ -47,6 +47,7 @@ Container environments:
 - [Rancher Desktop](https://rancherdesktop.io/): Kubernetes and container management to the desktop
 - [Colima](https://github.com/abiosoft/colima): Docker (and Kubernetes) on macOS with minimal setup
 - [Finch](https://github.com/runfinch/finch): Finch is a command line client for local container development
+- [Podman Desktop](https://podman-desktop.io/): Containers and Kubernetes for application developers
```

Comment:
Maybe it should be noted that Lima is supported as an optional extension

Reply:
Yes, that would be fair to mention - even if the extension is included by default

Reply:
In the marketing they separate between "Docker Desktop extension" and "Podman Desktop plug-in", so I guess it should be the same here - even if it is still packaged as an extension or "container runtime provider" or whatever.

https://podman-desktop.io/extend